qcodes/instrument_drivers/oxford/IPS120.py

```diff
@@ -0,0 +1,944 @@
+# OxfordInstruments_IPS120.py class, to perform the communication between the Wrapper and the device
+# Sjaak van Diepen <c.j.vandiepen@gmail.com>, 2017
+# Takafumi Fujita <t.fujita@tudelft.nl>, 2016
+# Mohammad Shafiei <m.shafiei@tudelft.nl>, 2011
+# Guenevere Prawiroatmodjo <guen@vvtp.tudelft.nl>, 2009
+# Pieter de Groot <pieterdegroot@gmail.com>, 2009
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
```

Comment:
@jenshnielsen Would it be an option to include the driver in qcodes with an explicit mention of the GPL in the driver directory? We have another driver that is GPL, and I do not think it is possible to relicense to MIT.